vllm/worker/tpu_worker.py

```diff
@@ -108,6 +108,8 @@ def determine_num_available_blocks(self) -> Tuple[int, int]:
                       torch.tensor([], dtype=torch.float32,
                                    device=self.device))
                      for _ in range(num_layers)]
+        bind_kv_cache(self.compilation_config.static_forward_context,
+                      [kv_caches])
```

Comment:
TPU kv caches are actually two separate tensors. would that matter here?

Reply:
It does not matter. The detailed structure of kv_cache in one layer (e.g., torch.Tensor/Tuple[torch.Tensor]) will not be accessed outside Attention backends.